### PR TITLE
Feature/structured data enhancement

### PR DESCRIPTION
**Pull request by AryaSvitkona**
### 1. Why is this change necessary?
Markup seems to be deprecated due to [Google documentation](https://developers.google.com/search/docs/advanced/structured-data/product#microdata). That causes a warning in test tool.

### 2. What does this change do, exactly?
Update markup according to guidelines.

### 3. Describe each step to reproduce the issue or behaviour.
1. Open Markup Check Tool by [Google](https://search.google.com/test/rich-results)
2. Provide product detail URL of a shopware instance
3. Run test 

### 4. Please link to the relevant issues (if any).
no relevant issue

### 5. Which documentation changes (if any) need to be made because of this PR?
no relevant issue

### 6. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code
- [x] I have read the contribution requirements and fulfil them.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,23 +5,28 @@ Please make sure to fulfil our contribution guideline (https://developers.shopwa
 -->
 
 ### 1. Why is this change necessary?
+Markup seems to be deprecated due to [Google documentation](https://developers.google.com/search/docs/advanced/structured-data/product#microdata). That causes a warning in test tool.
```
Comment (mitelg) [inline .github/PULL_REQUEST_TEMPLATE.md]: please revert the changes in this file. it is used as template and should not contain your specific issue description

**Review by mitelg — CHANGES_REQUESTED** [review]
- could you please have a look at the conflict? you may need a rebase of your branch :wink: 
- you also need to sign the CLA

**Comment by mitelg** [comment]
hey @AryaSvitkona 
thanks for your first contribution :slightly_smiling_face: :+1: 

I left some comments for you :wink:

**Comment by AryaSvitkona** [comment]
Thank you @mitelg 
Was a bit unsettled since this is my first contribution :)

**Comment by AryaSvitkona** [comment]
Since feature branch was created from "next" branch, this PR will be closed

**Pull request closed without merging**